chain/client/src/client.rs

```diff
@@ -1794,6 +1815,28 @@ impl Client {
             ?blocks_missing_chunks,
             ?orphans_missing_chunks)
         .entered();
+        if let Some(chunk_distribution) = &self.chunk_distribution_network {
+            if chunk_distribution.config.enabled {
+                crate::chunk_distribution_network::request_missing_chunks(
+                    blocks_missing_chunks,
+                    orphans_missing_chunks,
+                    chunk_distribution.clone(),
+                    &mut self.chain.blocks_delay_tracker,
+                    &self.shards_manager_adapter,
+                )
+            } else {
+                self.p2p_request_missing_chunks(blocks_missing_chunks, orphans_missing_chunks);
+            }
+        } else {
+            self.p2p_request_missing_chunks(blocks_missing_chunks, orphans_missing_chunks);
+        }
```

Comment:
nit:
```
       if let Some(chunk_distribution) = &self.chunk_distribution_network {
            if chunk_distribution.config.enabled {
                return crate::chunk_distribution_network::request_missing_chunks(
                    blocks_missing_chunks,
                    orphans_missing_chunks,
                    chunk_distribution.clone(),
                    &mut self.chain.blocks_delay_tracker,
                    &self.shards_manager_adapter,
                )
            }
        }
        self.p2p_request_missing_chunks(blocks_missing_chunks, orphans_missing_chunks);
```

Reply:
Addressed as part of https://github.com/near/nearcore/pull/10480/commits/2a4d18189d23cc4a4a8d72b248ac65b8d87006e0